tests/PHPStan/Analyser/NodeScopeResolverPhp7Test.php

```diff
@@ -4,7 +4,7 @@
 
 use PHPStan\Testing\TypeInferenceTestCase;
 
-class LooseConstComparisonPhp7Test extends TypeInferenceTestCase
+class NodeScopeResolverPhp7Test extends TypeInferenceTestCase
```

Comment:
Since the neon file is only
```
parameters:
	phpVersion: 70400 # PHP 7.4
```
we could use those TestCase for all the configured-phpVersion-related tests instead of creating a TestCase every time.